src/battle/overlay_12_0226BEC4.c

```diff
@@ -17,7 +17,7 @@ SysTask *ov12_0226BEC4(BattleSystem *bsys) {
 }
 
 void ov12_0226BEF0(SysTask *task) {
-    FreeToHeap(sub_0201F988(task));
+    FreeToHeap(SysTask_GetData(task));
```

Comment:
'GetData' feels like a bad name but idk if there's enough information to expand on it further

Reply:
It's just the pointer to the task's stored data (passed to SysTask_CreateOnXYZQueue), which can be literally anything.